cornflow_client/core/instance_solution.py

```diff
@@ -146,9 +146,11 @@ def to_excel(self, path: str):
 
         is_xl_type(path)
```

Comment:
I'm curious about this function. Does it raise an error if the path is not excel? Because from the name it looks as if it just returns True or False.
Also, above I see a warning *and* an exception being razed? Does it make sense to do both?

Reply:
It does raise an exception: 

```
if not any(ext in path for ext in [".xlsx", ".xls", ".xlsm"]):
        raise ConnectionError(
            "Excel_file argument should be a string with an excel extension"
        )
```

And yes you are right we do not need both a warning and an exception on module import. Will take one of them out.